plugin/trino-pinot/src/test/java/io/trino/plugin/pinot/AbstractPinotIntegrationSmokeTest.java

```diff
@@ -142,6 +142,8 @@ protected QueryRunner createQueryRunner()
         ImmutableList.Builder<ProducerRecord<String, GenericRecord>> allTypesRecordsBuilder = ImmutableList.builder();
         for (int i = 0, step = 1200; i < MAX_ROWS_PER_SPLIT_FOR_SEGMENT_QUERIES - 2; i++) {
             int offset = i * step;
+            long updatedAtMillis = initialUpdatedAt.plusMillis(offset).toEpochMilli();
+            String jsonObject = format("{\"offset\":%d,\"str\":\"abc\",\"arr\":[1,2,3]}", offset);
```

Comment:
inline since they are used once and I make the test data appear closer with all the other test data.